commander/src/bootstrapping/commands/genesis-block.ts

```diff
@@ -0,0 +1,233 @@
+/*
+ * LiskHQ/lisk-commander
+ * Copyright © 2021 Lisk Foundation
+ *
+ * See the LICENSE file at the top-level directory of this distribution
+ * for licensing information.
+ *
+ * Unless otherwise agreed in a custom licensing agreement with the Lisk Foundation,
+ * no part of this software, including this file, may be copied, modified,
+ * propagated, or distributed except according to the terms contained in the
+ * LICENSE file.
+ *
+ * Removal or modification of this copyright notice is prohibited.
+ *
+ */
+import { getAddressFromPassphrase } from '@liskhq/lisk-cryptography';
+import { createGenesisBlock, getGenesisBlockJSON, accountAssetSchemas } from '@liskhq/lisk-genesis';
+import { Account } from '@liskhq/lisk-chain';
+import * as cryptography from '@liskhq/lisk-cryptography';
+import { Application, PartialApplicationConfig } from 'lisk-framework';
+import { objects } from '@liskhq/lisk-utils';
+import { Command, flags as flagParser } from '@oclif/command';
+import fs from 'fs-extra';
+import { join, resolve } from 'path';
+import inquirer from 'inquirer';
+import { createMnemonicPassphrase } from '../../utils/mnemonic';
+import { defaultConfig } from '../../utils/config';
+
+interface AccountInfo {
+	readonly address: string;
+	readonly passphrase: string;
+}
+
+const createAccount = (): AccountInfo => {
+	const passphrase = createMnemonicPassphrase();
+	const address = getAddressFromPassphrase(passphrase).toString('hex');
+
+	return {
+		passphrase,
+		address,
+	};
+};
+
+export abstract class BaseGenesisBlockCommand extends Command {
+	static description = 'Creates genesis block file.';
+	static examples = [
+		'genesis-block:create --output mydir',
+		'genesis-block:create --output mydir --accounts 10',
+		'genesis-block:create --output mydir --accounts 10 --validators 103',
+		'genesis-block:create --output mydir --accounts 10 --validators 103 --token-distribution 500',
+	];
+
+	static flags = {
+		output: flagParser.string({
+			char: 'o',
+			description: 'Output folder path of the generated genesis block',
+			default: '',
+		}),
+		accounts: flagParser.integer({
+			char: 'a',
+			description: 'Number of non-validator accounts to generate',
+			default: 10,
+		}),
+		validators: flagParser.integer({
+			char: 'v',
+			description: 'Number of validator accounts to generate',
+			default: 103,
+		}),
+		'token-distribution': flagParser.integer({
+			char: 't',
+			description: 'Amount of tokens distributed to each account',
+			default: 500,
+		}),
+	};
+
+	async run(): Promise<void> {
+		const {
+			flags: { output, accounts, validators, 'token-distribution': tokenDistribution },
+			// eslint-disable-next-line @typescript-eslint/no-unsafe-assignment
+		} = this.parse(BaseGenesisBlockCommand);
+
+		const prepareNormalAccounts = (
```

Comment:
this function can be defined outside?
also, probably better to use default account as base, and check token module is registered or not